static/index.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Themeroller</title>
```

Comment:
This is picky, but maybe say Chassis Themeroller for the title?  

That's really my only feedback I have left on this. Tweak that and I'll pull this in.